libraries/chain/controller.cpp

```diff
@@ -2560,6 +2693,8 @@ const chainbase::database& controller::db()const { return my->db; }
 chainbase::database& controller::mutable_db()const { return my->db; }
 
 const fork_database& controller::fork_db()const { return my->fork_db; }
+b1::chain_kv::database& controller::kv_database() { return my->kv_database; }
+b1::chain_kv::undo_stack& controller::kv_undo_stack() { return my->kv_undo_stack; }
```

Comment:
create_kv_rocksdb_context() needs kv_database and kv_undo_stack as argutments but kv_database() and kv_undo_stack() aren't accessible because they are private.

Reply:
not following why @kimjh2005 thinks this, since the code does that and it compiles